modules/nuxtHubLanding/index.ts

```diff
@@ -0,0 +1,76 @@
+import {addServerHandler, createResolver, defineNuxtModule, logger, extendPages} from 'nuxt/kit'
+import defu from "defu";
+
+
+interface ModuleOptions {
+    appName?: string
+    verifyEmail?: boolean
+    email?: string,
+    resendApiKey?: string,
+}
+
+const log = logger.withTag('nuxtHubLanding')
+
+export default defineNuxtModule<ModuleOptions>({
+    meta: {
+        name: 'nuxtHubLanding',
+        configKey: "landing",
```

Comment:
Thought: I think if this is supposed to be published as module a more descriptive name is required. How about `evoize-waitlist`?